src/pallavi/assignment_10/TestCase10.java

```diff
@@ -0,0 +1,44 @@
+package pallavi.assignment_10;
+
+import java.util.HashMap;
+import java.util.concurrent.TimeUnit;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+/*Test case : 10
+
+Name -> "Maulik", print lastName and UserName*/
+public class TestCase10 {
+
+	static HashMap<String, String> getEmpDetails(WebDriver driver, String name) {
```

Comment:
if you create driver at instance level so we not require to pass reference of driver in every method